app/models/contact.rb

```diff
@@ -30,23 +30,22 @@ class Contact < ApplicationRecord
   belongs_to :account
   validates :phone,
             allow_blank: true,
-            format: { with: /\+[1-9]\d{1,14}\z/, message: "Número inválido" }
-
-
+            format: { with: /\+[1-9]\d{1,14}\z/ }
 
   has_many :deals, dependent: :destroy
   belongs_to :app, polymorphic: true, optional: true
 
-  def self.ransackable_attributes(auth_object = nil)
-    ["additional_attributes", "app_id", "app_type", "created_at", "custom_attributes", "email", "full_name", "id", "phone", "updated_at"]
+  def self.ransackable_attributes(_auth_object = nil)
+    %w[additional_attributes app_id app_type created_at custom_attributes email full_name id
+       phone updated_at]
   end
 
   def connected_with_chatwoot?
     additional_attributes['chatwoot_id'].present?
   end
 
-  FORM_FIELDS = [:full_name, :email, :phone, :label_list, :chatwoot_conversations_label_list]
-  after_commit :export_contact_to_chatwoot, on: [:create, :update]
+  FORM_FIELDS = %i[full_name email phone label_list chatwoot_conversations_label_list]
+  after_commit :export_contact_to_chatwoot, on: %i[create update]
```

Comment:
This change replaces a regular symbol array with the %i shorthand syntax. This is bad because it doesn’t follow the purpose of the PR.